indico/modules/events/registration/fields/choices.py

```diff
@@ -156,7 +156,10 @@ def _check_number_of_places(new_data):
                     places_used_dict.setdefault(k, 0)
                     if old_data and old_data.data:
                         places_used_dict[k] -= old_data.data.get(k, 0)
-                    places_used_dict[k] += new_data[k]
+                    try:
+                        places_used_dict[k] += new_data[k]
+                    except TypeError:
```

Comment:
Actually, I don't see how this can happen in v3.2+ since `new_data` is always guaranteed to be dict mapping strings to ints, and any `k` in here is actually one that's guaranteed to be a valid choice and not some random nonsense comint from the user.